src/cheri/zylevels1.adoc

```diff
@@ -0,0 +1,171 @@
+(both implicitly in loads from memory and explicitly with <<ACPERM>>).
+
+=== Interaction with <<infinite-cap,Infinite Capabilities>>
+
+As expected, <<infinite-cap,Infinite capabilities>> have both new permissions, <<zylevels1_lg_perm>> and <<zylevels_sl_perm>> set.
+Further, Infinite caps have <<zylevels1_gl_perm>> set to _global_.
+
+=== Interaction with <<ACPERM>> and <<GCPERM>>
+
+.Extended capability permissions bit field (see <<acperm_bit_field>>)
+[#zylevels1_acperm_bit_field]
+include::./img/zylevels1-acperm_bit_field.edn[]
+
+The <<zylevels1_gl_perm>>, <<zylevels1_sl_perm>>, and <<zylevels1_lg_perm>> fields
+are mapped into the <<acperm_bit_field,capability permissions bitfield>> (<<acperm_bit_field>>),
+used by <<ACPERM>> and <<GCPERM>>,
+as shown in <<zylevels1_acperm_bit_field>>.
+
+=== Interaction with <<LC>> and <<zalrsc_cheri>>
+
+As outlined above, Zylevels1 introduces two new constraints on capabilities loaded from memory,
+as part of a <<LC>> instruction `lc cd, offset(cs1)` or <<LR_C>> instruction `lr.c cd, offset(cs1)`.
+These may be phrased as "implicit <<ACPERM>>-s" performed on the loaded capability thus:
+
+* If `cd.tag=1`, `cd` *is not* sealed, and `cs1` does not grant <<zylevels1_lg_perm>>,
+  then an implicit <<ACPERM>> is performed, clearing both <<zylevels1_gl_perm>> and <<zylevels1_lg_perm>> of `cd`.
+
+* If `cd.tag=1`, `cd` *is* sealed, and `cs1` does not grant <<zylevels1_lg_perm>>,
+  then an implicit <<ACPERM>> is performed, clearing <<zylevels1_gl_perm>> of `cd`.
+
+NOTE: Missing <<zylevels1_lg_perm>> also affects the <<zylevels1_gl_perm>> of sealed capabilities,
+since notionally the latter is not a permission but rather a data flow label attached to the loaded value.
+
+NOTE: Because <<zylevels1_sl_perm>> is relevant only to capabilities bearing <<w_perm>>,
+the attenuation performed by a load whose authority (`cs1`) does not convey <<lm_perm>>
+will necessarily also clear <<zylevels1_sl_perm>>.
+
+=== Interaction with <<SC>> and <<zalrsc_cheri>>
+
+As outlined above, Zylevels1 introduces a new constraint on capabilities stored to memory,
+as part of a <<SC>> instruction `sc cs2, offset(cs1)` or <<SC_C>> instruction `sc.c cs2, offset(cs1)`:
+the written valid tag may be set _only if_ either
+
+* `cs2` 's <<zylevels1_gl_perm>> is set or
+* `cs1` 's <<zylevels1_sl_perm>> is set.
+
+NOTE: While <<zylevels1_lg_perm>> attenuates by reducing <<zylevels1_gl_perm>> and <<zylevels1_lg_perm>>,
+<<zylevels1_sl_perm>> attenuates by clearing valid tags.
+
+=== Interaction with <<SCSS>>
+
+Implementations of Zylevels1 must ensure that a <<SCSS>> instruction `scss rd, cs1, cs2`
+indicates that `cs1` is a subset of `cs2` (that is, sets `rd` to 1) only if either
+
+* `cs2` 's <<zylevels1_gl_perm>> is clear or
+* `cs1` 's <<zylevels1_gl_perm>> is set.
+
+The existing permission subset logic applies to the new <<zylevels1_sl_perm>> and <<zylevels1_lg_perm>>.
```

Comment:
Is the GL bit irrelevant to SCSS? If so, it might be worth to state that here. In general, I am wondering whether we would want a paragraph somewhere earlier differentiating between the permissions and the GL bit. What do you think?

Reply:
I (attempted to) keep the existing semantics.  In particular, the `GL` bit is handled by lines 129-130, I think?